exporter/awsemfexporter/util.go

```diff
@@ -26,9 +26,10 @@ import (
 )
 
 var patternKeyToAttributeMap = map[string]string{
-	"ClusterName": "aws.ecs.cluster.name",
-	"TaskId":      "aws.ecs.task.id",
-	"NodeName":    "k8s.node.name",
+	"ClusterName":         "aws.ecs.cluster.name",
+	"TaskId":              "aws.ecs.task.id",
+	"NodeName":            "k8s.node.name",
+	"ContainerInstanceId": "aws.ecs.container.instance.id",
```

Comment:
I think instance is not directly coupled to ECS. I don't think prefixing `instance.id` with `aws.ecs` is meaningful. Same goes for cluster though.

Reply:
It makes sense, maybe it's better to use `aws.container.instance.id`. 
Also for the cluster, it seems that the name is defined by you, do you want me to change it?

Reply:
Oh my bad. If it's ECS Container Instance ID, then the name is fine. For EKS, I believe its NodeName.

Reply:
Got it, thanks